modules/level_set/src/base/LevelSetTypes.C

```diff
@@ -0,0 +1,14 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+// Level set includes
+#include "LevelSetTypes.h"
+
+// MOOSE includes
+#include "MooseEnumItem.h"
+
+const ExecFlagType LevelSet::EXEC_ADAPT_MESH("ADAPT_MESH", 8000);
```

Comment:
Perhaps.

Reply:
Would still like to see this addressed. It's one of the weird unfinished things since it requires the user to produce something and they may not know how to do that.